### PR TITLE
[Fundamentals] Refer Zaraz on Google Analytics page

### PR DESCRIPTION
**Pull request by jrencz**
- Disclaimer in first sentence has been refined
- pre-existing content has been wrapped into h2 `Standard GA setup`
- new section under header `Zaraz`

My intention is to improve discoverability of Zaraz

### CODE REVIEW

File: content/fundamentals/get-started/reference/google-analytics.md [hunk 1]
```diff
@@ -18,3 +20,13 @@ Cloudflare only affects analytics tools that read logs directly from your web se
 To troubleshoot potential issues with Google Analytics, refer to [Common GA setup mistakes](https://support.google.com/analytics/answer/1009683).
 
 {{</Aside>}}
+
+## Zaraz
+
+As an alternative to the standard setup of Google Analytics with tag/snippet, Cloudflare offers a way to use Google Analytics with [Zaraz](https://developers.cloudflare.com/zaraz/). Zaraz is a solution that allows Google Analytics to collect data without its script loaded on the website. If GA is set up this way, then not all features may be available. 
+
+{{<Aside type="note" header="Binding">}}
```
Comment (jrencz) [inline content/fundamentals/get-started/reference/google-analytics.md]:
```suggestion
{{<Aside type="note">}}
```

File: content/fundamentals/get-started/reference/google-analytics.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ meta:
 
 # Using Google Analytics with Cloudflare
 
-Using Cloudflare does not affect Google Analytics (GA) tracking.
+Using Cloudflare does not affect Google Analytics (GA) tracking if it's added to the website [in one of ways recommended by Google](https://support.google.com/analytics/answer/9304153#add-tag).
```
Comment (kodster28) [inline content/fundamentals/get-started/reference/google-analytics.md]:
```suggestion
Using Cloudflare does not affect Google Analytics (GA) tracking if it is added to the website [in one of ways recommended by Google](https://support.google.com/analytics/answer/9304153#add-tag).
```

File: content/fundamentals/get-started/reference/google-analytics.md [hunk 3]
```diff
@@ -18,3 +20,13 @@ Cloudflare only affects analytics tools that read logs directly from your web se
 To troubleshoot potential issues with Google Analytics, refer to [Common GA setup mistakes](https://support.google.com/analytics/answer/1009683).
 
 {{</Aside>}}
+
+## Zaraz
+
+As an alternative to the standard setup of Google Analytics with tag/snippet, Cloudflare offers a way to use Google Analytics with [Zaraz](https://developers.cloudflare.com/zaraz/). Zaraz is a solution that allows Google Analytics to collect data without its script loaded on the website. If GA is set up this way, then not all features may be available.
```
Comment (kodster28) [inline content/fundamentals/get-started/reference/google-analytics.md]:
```suggestion
As an alternative to the standard setup of Google Analytics with tag/snippet, Cloudflare offers a way to use Google Analytics with [Zaraz](/zaraz/). Zaraz is a solution that allows Google Analytics to collect data without its script loaded on the website. If GA is set up this way, then not all features may be available. 
```

File: content/fundamentals/get-started/reference/google-analytics.md [hunk 4]
```diff
@@ -18,3 +20,13 @@ Cloudflare only affects analytics tools that read logs directly from your web se
...
 
 {{</Aside>}}
+
+## Zaraz
+
+As an alternative to the standard setup of Google Analytics with tag/snippet, Cloudflare offers a way to use Google Analytics with [Zaraz](https://developers.cloudflare.com/zaraz/). Zaraz is a solution that allows Google Analytics to collect data without its script loaded on the website. If GA is set up this way, then not all features may be available. 
+
+{{<Aside type="note">}}
+
+Details about features of Google Analytics that are unavailable with Zaraz can be found in [Zaraz FAQ](https://developers.cloudflare.com/zaraz/faq/#tools)
```
Comment (kodster28) [inline content/fundamentals/get-started/reference/google-analytics.md]:
```suggestion
Details about features of Google Analytics that are unavailable with Zaraz can be found in [Zaraz FAQ](zaraz/faq/#tools)
```

**Review by kodster28 — APPROVED** [review]

**Comment by kodster28** [comment]
Thanks for the contribution, @jrencz!

File: content/fundamentals/get-started/reference/google-analytics.md [hunk 5]
```diff
@@ -18,3 +20,13 @@ Cloudflare only affects analytics tools that read logs directly from your web se
...
 
 {{</Aside>}}
+
+## Zaraz
+
+As an alternative to the standard setup of Google Analytics with tag/snippet, Cloudflare offers a way to use Google Analytics with [Zaraz](/zaraz/). Zaraz is a solution that allows Google Analytics to collect data without its script loaded on the website. If GA is set up this way, then not all features may be available. 
+
+{{<Aside type="note">}}
+
+Details about features of Google Analytics that are unavailable with Zaraz can be found in [Zaraz FAQ](zaraz/faq/#tools)
```
Comment (kodster28) [inline content/fundamentals/get-started/reference/google-analytics.md]:
```suggestion
Details about features of Google Analytics that are unavailable with Zaraz can be found in [Zaraz FAQ](/zaraz/faq/#tools)
```

Comment (jrencz) [reply content/fundamentals/get-started/reference/google-analytics.md]: I was about to ask what can I do to fix the check, but you got it faster :)

**Pull request merged**